src/main/java/com/spring/restaurant/service/EmployeeMenuImpl.java

```diff
@@ -0,0 +1,17 @@
+package com.spring.restaurant.service;
+
+import com.spring.restaurant.dao.EmployeeMenu;
+import com.spring.restaurant.repository.EmployeeMenuRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EmployeeMenuImpl {
```

Comment:
Avoid using Impl at end of class names if you are not implementing any interfaces